storage/badger/dkg_keys.go

```diff
@@ -66,12 +66,30 @@ func (k *DKGKeys) retrieveTx(epochCounter uint64) func(tx *badger.Txn) (*dkg.DKG
 	}
 }
 
+// InsertMyDKGPrivateInfo insert the DKG private key to database when DKG was completed, and a DKG private key was successfully generated.
 func (k *DKGKeys) InsertMyDKGPrivateInfo(epochCounter uint64, info *dkg.DKGParticipantPriv) error {
```

Comment:
I know you only added to this implementation. Nevertheless, it would be great if you could document possible error returns here as well as in the interface https://github.com/onflow/flow-go/blob/df5e576c4203cbcd559adc4b47a5466a8f78bbaa/storage/dkg_keys.go#L8-L9
that would be much appreciated 🙏 🙇